src/azul/openapi/schema.py

```diff
@@ -192,16 +196,67 @@ def enum(*items: PrimitiveJSON, type_: TYPE = None) -> JSON:
             "bar"
         ]
     }
-    """
 
+    >>> assert_json(enum(items=['a']))
+    {
+        "type": "string",
+        "enum": [
+            "a"
+        ]
+    }
+
+    >>> assert_json(enum('a', items=['b']))
+    Traceback (most recent call last):
+    ...
+    azul.RequirementError: Cannot specify positional arguments and the `items` keyword argument
+
+    >>> enum(items={1: 2, 'foo': 3})
+    Traceback (most recent call last):
+    ...
+    AssertionError: ([1, 'foo'], <class 'int'>)
+
+    >>> assert_json(enum(items={1: 2, 2: 3}))
```

Comment:
Why are we passing a dictionary if the values in the dictionary entries are discarded?

Reply:
Closed #2375